docs/tutorial/part06.rst

```diff
@@ -2,5 +2,648 @@
+                'nodes_related': self.serialize_relationships(self.entities.all(), 'OFFICER_OF'),
+            },
+            {
+                'nodes_type': 'Officer',
+                'nodes_related': self.serialized_realtionships_of_type('Officer'),
+            },
+        ]
+
+Other::
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'Officer',
+                'nodes_related': self.serialized_realtionships_of_type('Officer'),
+            },
+            {
+                'nodes_type': 'Entity',
+                'nodes_related': self.serialized_realtionships_of_type('Entity'),
+            },
+            {
+                'nodes_type': 'Address',
+                'nodes_related': self.serialize_relationships(self.addresses.all(), 'REGISTERED_ADDRESS'),
+            },
+        ]
+
+
+Call the serialize relationships methods on the helpers
+=======================================================
+
+On your ``fetch_api/utils.py`` the ``fetch_node_details`` function change put this above the return
+statement::
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+So the function should be looking something like this::
+
+    def fetch_node_details(node_info):
+        node_type       = node_info['node_type']
+        node_id         = node_info['node_id']
+        node            = MODEL_ENTITIES[node_type].nodes.get(node_id=node_id)
+        node_details    = node.serialize
+
+        # Make sure to return an empty array if not connections
+        node_details['node_connections'] = []
+        if (hasattr(node, 'serialize_connections')):
+            node_details['node_connections'] = node.serialize_connections
+
+        return node_details
+
+Return the json to the frontend
+===============================
+
+Now if you call your functions ``fetch_node_details`` and ``fetch_nodes`` should be returning the
+same data but in a way that is JSON parsable, so let's change a couple things in order of returning
+this data that we need.
+
+In your project settings file for us it is ``paradise_papers_search/settings/base.py`` but if you
+are using a costume project it may be different for you, add the following::
+
+    # Rest-Framework settings
+    REST_FRAMEWORK = {
+        'DEFAULT_RENDERER_CLASSES': (
+            'rest_framework.renderers.JSONRenderer',
+        )
+    }
+
+This is going to cause that when we invoke the render method on our APIs views the method will call
+the JSON renderer rather than just the HTTP one.
+
+So once again we will clean our ``fetch_api/views.py`` file and leave it like this::
+
+    from rest_framework.views import APIView
+    from rest_framework.response import Response
+    from .models.helpers import fetch_nodes

```

Comment:
Before this part at the tutorial, there's not a definition or process to create a module named helpers. It should be added.

Reply:
We change the `helpers` file to `utils`
https://github.com/neo4j-examples/paradise-papers-django/blob/master/paradise_papers_search/fetch_api/utils.py